cmd/version/version.go

```diff
@@ -60,23 +66,62 @@ func getVersion(ctx context.Context, args []string, cmdCtx cmdCore.CommandContex
 		return err
 	}
 
+	getControlePlaneVersion(ctx, cmdCtx)
+	return nil
+}
+
+func printVersion(response versionOutput) error {
+	b, err := json.MarshalIndent(response, "", "  ")
+	if err != nil {
+		return fmt.Errorf("err %v: ", err)
```

Comment:
Unit test covering this branch and other branches in this file

Reply:
meaning ?